tests/e2e/test_arxiv_harvest.py

```diff
@@ -228,3 +180,37 @@ def _all_in_status(status):
         interaction_name='ticket_new',
         times=1,
     )
+
+
+@with_mitmproxy
+def test_harvest_nucl_th_and_jlab_curation(inspire_client, mitm_client):
+    inspire_client.e2e.schedule_crawl(
+        spider='arXiv_single',
+        workflow='article',
+        url='http://export.arxiv.org/oai2',
+        identifier='oai:arXiv.org:1806.05669',  # nucl-th record
+    )
+
+    halted_entry = wait_for(lambda: _all_in_status(inspire_client, 'COMPLETED'))
```

Comment:
Shouldn't it wait for jlab curation?

Reply:
That comes later xd, we should probably extend the test to do the actual jlab floww

Reply:
YEah, the `edit_article` workflow here is not triggered at all, you have to start it from the ticket. In this case, I would like to check the ticket content

Reply:
I think we sholud test the whole flow

Reply:
DONE :sunglasses: